kava-stats.js

```diff
@@ -691,39 +660,54 @@ const setRewardsApyDisplayValues = async (denoms, siteData, cssIds) => {
 const setTotalAssetsSuppliedDisplayValue = async (siteData, cssIds) => {
   let cssId = cssIds['totalAssetsSupplied'];
   let totalAssetsSupplied = 0;
-  const totalSuppliedData = siteData['totalSuppliedData'];
-  for (const denom in totalSuppliedData) {
-    const denomSuppliedUsd = totalSuppliedData[denom]
-    totalAssetsSupplied += denomSuppliedUsd
-  }
+  const platformAmounts = siteData['platformAmounts'];
+  const denomConversions = siteData['denomConversions'];
+  const prices = siteData['prices']; 
+  for (const denom in platformAmounts) {
+    // TEMPORARY FIX
+    if (denom === 'busd-b') continue;
+    let price = 1; 
+    if(prices[denom]) price = prices[denom].price; 
```

Comment:
BUSD-B is a cdp type that uses the BUSD denom. We can use the same price for BUSD whether it is a BUSD-A or BUSD-B cdp